Bottom: 048241532db126b67555a1dc88bd858f098ba3ae
Top:    45b87c582bffb2e5c01b0742ffd6da1fd1ae944d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-05 15:42:56 +0100

Refresh of tag-vnodes-with-details-of

---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index 4343e82..af21e3f 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -177,8 +177,8 @@ thindraw_contour (hidGC gc, PLINE *pl)
         {
           Angle start_angle, end_angle, delta_angle;
 
-          start_angle = 180. - TO_DEGREES (atan2 ((v->prev->point[1] - v->cy), (v->prev->point[0] - v->cx)));
-          end_angle   = 180. - TO_DEGREES (atan2 ((      v->point[1] - v->cy), (      v->point[0] - v->cx)));
+          start_angle = TO_DEGREES (atan2 ((v->prev->point[1] - v->cy), -(v->prev->point[0] - v->cx)));
+          end_angle   = TO_DEGREES (atan2 ((      v->point[1] - v->cy), -(      v->point[0] - v->cx)));
           delta_angle = end_angle - start_angle;
 
           if (delta_angle > 180.) delta_angle -= 360.;
